Mathlib/Data/FunLike/Basic.lean

```diff
@@ -211,6 +215,8 @@ variable {F α β : Sort*} [i : FunLike F α fun _ ↦ β]
 
 namespace FunLike
 
+instance (priority := 110) ndHasCoeToFun : CoeFun F fun _ ↦ α → β where coe := FunLike.coe
```

Comment:
```suggestion
/-- Non-Dependent variant of `FunLike.hasCoeToFun`.

We give this higher priority because it is simpler and therefore significantly speeds up usages of `FunLike`. -/
instance (priority := 110) ndHasCoeToFun : CoeFun F fun _ ↦ α → β where coe := FunLike.coe
```

Reply:
I think our comments crossed, https://github.com/leanprover-community/mathlib4/pull/7905#discussion_r1372940923